lib/ballot/proposer_transaction.go

```diff
@@ -96,7 +96,6 @@ func NewProposerTransactionFromBallot(blt Ballot, opc operation.CollectTxFee, op
 
 var ProposerTransactionWellFormedCheckerFuncs = []common.CheckerFunc{
 	transaction.CheckOverOperationsLimit,
-	transaction.CheckSequenceID,
```

Comment:
We don't need to check sequenceID ?

Reply:
In `CheckSequenceID` method,
https://github.com/bosnet/sebak/pull/632/files/1313fa1f5b5df1689bb8a910037ad17c5e853158#diff-d7e465cfa868d70ec0209456513c57d4L43

The checking part is https://github.com/bosnet/sebak/pull/632/files#diff-3ac5aa21aabcbf6c057cb872f5980636R217